CHANGELOG.md

```diff
@@ -56,6 +56,7 @@
 * [FEATURE] Added `-api.http-request-headers-to-log` allowing for the addition of HTTP Headers to logs #4803
 * [FEATURE] Distributor: Added a new limit `-validation.max-labels-size-bytes` allowing to limit the combined size of labels for each timeseries. #4848
 * [FEATURE] Storage/Bucket: Added `-*.s3.bucket-lookup-type` allowing to configure the s3 bucket lookup type. #4794
+* [FEATURE] QueryFrontend: Implement experimental vertical sharding at query frontend for range/instance queries. #4863
```

Comment:
typo s/instance/instant/g

Reply:
Also, can should we update https://cortexmetrics.io/docs/configuration/v1guarantees/#experimental-features ?

Reply:
done